typescript/api.ts

```diff
@@ -60,10 +97,7 @@ export type APIDeploymentPayload = {
   variables: {
     [key: string]: string
   }
-  routes: Array<{
-    module: string
-    route: string
-  }>
+  routes: APIEnvironmentRoute[]
```

Comment:
should this be optional??

Reply:
it is currently required and will require more changes the API to make it optional. No breaking changes and no work to update is prefereable to no breaking changes and work to update IMO.